packages/platform/src/response.rs

```diff
@@ -1,12 +1,31 @@
+use serde::Serialize;
+
 use sdk::{
-    cosmwasm_ext::Response,
-    cosmwasm_std::{to_binary, StdResult},
+    cosmwasm_ext::Response as CwResponse,
+    cosmwasm_std::{to_binary, Binary, StdResult},
 };
-use serde::Serialize;
 
-pub fn response<T>(resp: &T) -> StdResult<Response>
+#[inline]
+pub fn response<T>(response: &T) -> StdResult<Response>
```

Comment:
[here and below] let's keep the `Std` prefix for the Rust Std namespace and pertaining to that space types. For uniformity let's use the `Cw` prefix.